Twitter/Contracts/V1/Responses/AuthResponse.cs

```diff
@@ -1,8 +1,12 @@
-﻿namespace Twitter.Contracts.V1.Responses
+﻿using System.Collections.Generic;
+using Twitter.Domain;
+
+namespace Twitter.Contracts.V1.Responses
```

Comment:
It will be helpful if you read about File Scoped Namespaces involved in C# 10 (.Net 6). You don't need to change but you can learn something new :)

Reply:
Or click at line 4 and VS will give you suggestion for refactoring to Convert to File scoped namespace. But you don't need to change.

Reply:
I've read it, thank you